plugins/PostProcessingPlugin/scripts/MultiExtColorMix.py

```diff
@@ -0,0 +1,623 @@
+                },
+                "park_head":
+                {
+                    "label": "Park and Purge",
+                    "description": "Whether to Park the head and Purge before starting a new Mix Ratio.",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": true
+                },
+                "park_x":
+                {
+                    "label": "    Park X Location",
+                    "description": "What X.",
+                    "type": "int",
+                    "default_value": 0,
+                    "enabled": "park_head"
+                },
+                "park_y":
+                {
+                    "label": "    Park Y Location",
+                    "description": "What Y.",
+                    "type": "int",
+                    "default_value": "0",
+                    "enabled": "park_head"
+                },
+                "purge_amt":
+                {
+                    "label": "    Purge Amount",
+                    "description": "The extruder to use after mixing ends.  The mixing extruder is 'Extruder_Count +1 (will be T3 in the gcode).",
+                    "type": "int",
+                    "unit": "mm ",
+                    "default_value": 25,
+                    "minimum_value": "1",
+                    "maximum_value": "50",
+                    "enabled": "park_head"
+                }
+            }
+        }"""
+    
+    def execute(self, data):
+        MyCura = Application.getInstance().getGlobalContainerStack()   
+        extruder_count = MyCura.getProperty("machine_extruder_count", "value")
+        print_sequence = str(MyCura.getProperty("print_sequence", "value"))
+        init_extruder = Application.getInstance().getExtruderManager().getInitialExtruderNr()
+        if extruder_count >1:
+            T0_include = self.getSettingValueByKey("T0_include")
+            T1_include = self.getSettingValueByKey("T1_include")
+            T2_include = False
+        if extruder_count == 3:
+            T2_include = self.getSettingValueByKey("T2_include")
+        enabled_extruders = extruder_count
+        park_head = bool(self.getSettingValueByKey("park_head"))
+        park_x = str(self.getSettingValueByKey("park_x"))
+        park_y = str(self.getSettingValueByKey("park_y"))
+        purge_amt = str(self.getSettingValueByKey("purge_amt"))
+        park_string = ""
+        
+        # If it is a single extruder printer or if only one extruder is enabled then exit with a message-----------
+        if extruder_count == 1 or enabled_extruders < 2:
+            data[0] += ";  Multi-Extruder Color Mixer did not run because - Single extruder printer or single extruder enabled\n"
+            Message(text = "Color Mixer: {}".format("The post processor exited: single extruder printer OR single extruder enabled.")).show()
+            return data
```

Comment:
Since these messages are different it is more descriptive to separate the messages
```suggestion
        if extruder_count == 1:
            data[0] += ";  Multi-Extruder Color Mixer did not run because - selected printer only supports a printing on a single extruder\n"
            Message(text = "Color Mixer: {}".format("The post processor exited: selected printer only supports a printing on a single extruder.")).show()
            return data
        if enabled_extruders < 2:
            data[0] += ";  Multi-Extruder Color Mixer did not run because an insufficient number of extruder is enabled\n"
            Message(text = "Color Mixer: {}".format("The post processor exited: insufficient number of extruder are enabled.")).show()
```